pages/index.jsx

```diff
@@ -58,6 +58,7 @@ export async function getServerSideProps(context) {
 
   return {
     props: { topics }, // will be passed to the page component as props
+    revalidate: 60 * 30, // In seconds
```

Comment:
In seconds is not a good comment because other developers may still confuse what's the total cache expiry would be. A better approach is to define a meaningful const for the naming.

```javascript
const HALF_HOUR = 60 * 30;

return {
  // ...
  revalidate: HALF_HOUR
}
```